src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -310,6 +313,13 @@ private void run(Runnable task) {
             }
         }
     }
+    @Hidden
```

Comment:
Missing line break.
```suggestion

    @Hidden
```

Reply:
Also include the comment:
```suggestion

    /**
     * The VM recognizes this method as special, so any changes to the
     * name or signature require corresponding changes in
     * JVM_FindScopedValueBindings().
     */
    @Hidden
```